rakelib/openhab.rake

```diff
@@ -10,7 +10,7 @@ require 'digest/md5'
 # Disabled due to part of buid / potentially refactor into classes
 namespace :openhab do
   @openhab_version = ENV['OPENHAB_VERSION'] || '3.0.2'
```

Comment:
should you also update the default openhab version to 3.1.0?

Reply:
yes, but i will push that as a different PR.